### DISCUSSION TITLE
Don't store binary files to git history

### DISCUSSION THREAD

**Discussion by ArjixWasTaken** [Malcolm-Q/LC-LateGameUpgrades#353] [General]
As the title says, you shouldn't really store binary files (dll, exe, etc) in the repo's history.

Use GitHub releases for that.
Static assets like images or prefabs are acceptable to be in the git history, but having SVGs and yaml prefabs (if that is a thing) would be better.

Git is made for storing text, not binary data.

Binaries cannot be diffed, every time you edit a binary file, you are essentially discarding the old one and adding the new one to the history.

That makes the size of your repository increase a lot.

(If I am wrong about any of the above, feel free to call me out)

**Comment by WhiteSpike** [comment]
The output folder is an artifact of old ways. I will remove it when I get free time to touch this again.
